backend/dataline/services/conversation.py

```diff
@@ -140,7 +145,18 @@ async def query(
                 model_name=user_with_model_details.preferred_openai_model,
             ),
             history=history,
-        )
+            streaming=True,
+        ):
+            (message, result) = chunk
```

Comment:
Went for chunk_messages because it's messages from this current chunk. message_chunks sounds like multiple chunks from one single message hahah. Ditto for result